RecoTauTag/RecoTau/interface/AntiElectronIDMVA6.h

```diff
@@ -103,8 +129,12 @@ class AntiElectronIDMVA6 {
                   const antiElecIDMVA6_blocks::TauGammaMoms& tauGammaMoms,
                   const antiElecIDMVA6_blocks::ElecVars& elecVars);
 
+  double MVAValuePhase2(const antiElecIDMVA6_blocks::TauVars& tauVars,
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.

Reply:
Do you mean `TauVars`? It is name of structure so I think it is fine use capitalisation for it. NB. It is not new to this development.

Reply:
`MVAValuePhase2` is a data member and it should start with lower case

Reply:
Thanks, I got it - you mean name of the method. It follows convention used in this class previously that methods returning MVA values are called with capitalised `MVA` - I will change it consistently for all methods of this type.